benchmark-scripts/Makefile

```diff
@@ -17,4 +17,10 @@ run:
 	docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`/results:/tmp/results --net=host --privileged benchmark:dev /bin/bash
```

Comment:
Add copyright header to Makefile

# Copyright © 2024 Intel Corporation. All rights reserved.
# SPDX-License-Identifier: Apache-2.0